scrypto/src/component/stubs.rs

```diff
@@ -27,12 +28,16 @@ extern_blueprint_internal! {
     },
     {
         fn free(&mut self) -> Bucket;
-        fn lock_fee(&mut self, amount: Decimal);
+        fn lock_fee(&mut self, amount: Decimal) -> ();
```

Comment:
Ideally we'd remove the `-> ()` in this case

Reply:
But we can do that later :)

Reply:
I believe our python script can do it, doing it now.